AppAuth.podspec

```diff
@@ -31,7 +31,7 @@ It follows the OAuth 2.0 for Native Apps best current practice
   #       classes of AppAuth with tokens on watchOS and tvOS, but currently the
   #       library won't help you obtain authorization grants on those platforms.
 
-  s.ios.deployment_target = "7.0"
+  s.ios.deployment_target = "9.0"
   s.osx.deployment_target = "10.9"
```

Comment:
Probably need 10.12 here to be consistent with 2015 releases.

Reply:
The `libarclite_iphoneos` issue reported in the comment on https://github.com/openid/AppAuth-iOS/issues/759#issuecomment-1487880042 reproduces on macOS when the deployment target is < macOS 10.11 so this should be at least 10.11 to actually fix that linker issue, if not higher.

Reply:
Thanks for the catch. I have updated this. Podspec is linting locally. Publishing this PRs draft.